verilog/analysis/verilog_project.cc

```diff
@@ -185,6 +186,25 @@ bool VerilogProject::RemoveRegisteredFile(
   return false;
 }
 
+std::string VerilogProject::GetRelativePathToSource(
+    const absl::string_view absolute_filepath) {
+  // TODO add check if the absolute_filepath is out of the VerilogProject
+  std::filesystem::path absolutepath{absolute_filepath.begin(),
+                                     absolute_filepath.end()};
+  auto root = TranslationUnitRoot();
+  std::filesystem::path projectpath{root.begin(), root.end()};
+  auto relative = std::filesystem::relative(absolutepath, projectpath);
```

Comment:
Just a remark that we might consider revisiting the way we're resolving URIs at some point.

Right now, we essentially snip off the `file://` prefix, then implement the rest with standard with `std::filesystem` operations
(filesystem::relative, filesystem::absolute etc.): I suspect we might need to revisit that later as there are editors that might only have access files with some RPC mechanism or have some sort-of in-memory representation. So even a `file://` prefix might mean something else to them.

So at some point we might need some pluggable implementation that works on top of URIs (resolve relative, get content etc.). I think for now, the current implementation is ok, but it is worthwhile already thinking about some abstraction that goes beyond 'making filename from uri by removing prefix'.